src/content/docs/reference/Buffers/texture_format.mdx

```diff
@@ -8,115 +8,138 @@ sidebar:
+| -------------- | ---------- | -------------- | ----------------------------- | ----- |
+| `RGBA32UI`     | 4×32       | `RGBA_INTEGER` | `UNSIGNED_INT`                | ✔️     |
+| `RGBA16UI`     | 4×16       | `RGBA_INTEGER` | `UNSIGNED_SHORT`              | ✔️     |
+| `RGBA8UI`      | 4×8        | `RGBA_INTEGER` | `UNSIGNED_BYTE`               | ✔️     |
+| ~`RGB10_A2UI`~ | 10/10/10/2 | `RGBA_INTEGER` | `UNSIGNED_INT_2_10_10_10_REV` | ✔️     |
+| `RGB32UI`      | 3×32       | `RGB_INTEGER`  | `UNSIGNED_INT`                | ❌     |
+| `RGB16UI`      | 3×16       | `RGB_INTEGER`  | `UNSIGNED_SHORT`              | ❌     |
+| `RGB8UI`       | 3×8        | `RGB_INTEGER`  | `UNSIGNED_BYTE`               | ❌     |
+| `RG32UI`       | 2×32       | `RG_INTEGER`   | `UNSIGNED_INT`                | ✔️     |
+| `RG16UI`       | 2×16       | `RG_INTEGER`   | `UNSIGNED_SHORT`              | ✔️     |
+| `RG8UI`        | 2×8        | `RG_INTEGER`   | `UNSIGNED_BYTE`               | ✔️     |
+| `R32UI`        | 32         | `RED_INTEGER`  | `UNSIGNED_INT`                | ✔️     |
+| `R16UI`        | 16         | `RED_INTEGER`  | `UNSIGNED_SHORT`              | ✔️     |
+| `R8UI`         | 8          | `RED_INTEGER`  | `UNSIGNED_BYTE`               | ✔️     |
+
+### Signed Integral Formats
+
+Signed integral buffers store signed integers, just like the tin says. These are standard two's complement integers, which means they can store negative values.
+
+| Image Format | Bit Depth | Pixel Format   | Pixel Type | Image |
+| ------------ | --------- | -------------- | ---------- | ----- |
+| `RGBA32I`    | 4×32      | `RGBA_INTEGER` | `INT`      | ✔️     |
+| `RGBA16I`    | 4×16      | `RGBA_INTEGER` | `SHORT`    | ✔️     |
+| `RGBA8I`     | 4×8       | `RGBA_INTEGER` | `BYTE`     | ✔️     |
+| `RGB32I`     | 3×32      | `RGB_INTEGER`  | `INT`      | ❌     |
+| `RGB16I`     | 3×16      | `RGB_INTEGER`  | `SHORT`    | ❌     |
+| `RGB8I`      | 3×8       | `RGB_INTEGER`  | `BYTE`     | ❌     |
+| `RG32I`      | 2×32      | `RG_INTEGER`   | `INT`      | ✔️     |
+| `RG16I`      | 2×16      | `RG_INTEGER`   | `SHORT`    | ✔️     |
+| `RG8I`       | 2×8       | `RG_INTEGER`   | `BYTE`     | ✔️     |
+| `R32I`       | 32        | `RED_INTEGER`  | `INT`      | ✔️     |
+| `R16I`       | 16        | `RED_INTEGER`  | `SHORT`    | ✔️     |
+| `R8I`        | 8         | `RED_INTEGER`  | `BYTE`     | ✔️     |
+
+## Additional Pixel Types
+
+The shader loader supports some additional pixel types not directly associated with supported image formats. See the [OpenGL Wiki](https://www.khronos.org/opengl/wiki/Pixel_Transfer#Pixel_type) for more information about these.
+
+| Pixel Type                   |
+| ---------------------------- |
+| `UNSIGNED_BYTE_3_3_2`        |
+| `UNSIGNED_SHORT_5_6_5`       |
+| `UNSIGNED_SHORT_4_4_4_4`     |
+| `UNSIGNED_SHORT_5_5_5_1`     |
+| `UNSIGNED_INT_8_8_8_8`       |
+| `UNSIGNED_INT_8_8_8_8_REV`   |
+| `UNSIGNED_INT_10_10_10_2`    |
```

Comment:
There's probably no point in including these if they aren't supported/used by Iris

Reply:
> There's probably no point in including these if they aren't supported/used by Iris

The thing is that they _are_ supported, which not all pixel types are. They can still be used when loading raw custom textures if the raw data is stored with that pixel type afaik